ci/ci.yml

```diff
@@ -2,7 +2,7 @@ version: "3.7"
 services:
   quesma:
     build: ../quesma
-    image: quesma:latest
+    image: quesma:nigthly
```

Comment:
Not 100% sure about it, but I think this is incorrect. 

In our tests pipeline we build `quesma:latest` (here: https://github.com/QuesmaOrg/quesma/blob/260002c19ac40d032293051ea8fecdf72f311e11/.github/workflows/pipeline.yml#L124-L140) and then we start up `ci/ci.yml` with that image (https://github.com/QuesmaOrg/quesma/blob/260002c19ac40d032293051ea8fecdf72f311e11/.github/workflows/pipeline.yml#L168). Wouldn't that launch a released nightly image instead of just built `quesma:latest`?

Reply:
OK, now I see that `ci/ci.yml` always rebuilds it (because it uses `quesma:nightly` instead of `quesma/quesma:nightly`).

Reply:
Yea but it's all local - hopefully sometime in the future we'll become an official image (instead of `quesma/quesma`) and this becomes a concern 😉